java/test/src/main/java/test/IceBox/configuration/TestServiceI.java

```diff
@@ -10,8 +10,6 @@
 public class TestServiceI implements Service {
     @Override
     public void start(String name, Communicator communicator, String[] args) {
-        communicator.getProperties().setProperty("Ice.Package.Test", "test.IceBox.configuration");
```

Comment:
This was the reason I discovered #3943.

Thankfully though, this test doesn't actually unmarshal any classes or exceptions.
So the 'correct fix' was to just delete this property being set. As I also did in many other tests.